src/memgraph.cpp

```diff
@@ -99,9 +99,9 @@
 #include "audit/log.hpp"
 #endif
 
-constexpr const char *kMgUser = "MG_USER";
-constexpr const char *kMgPassword = "MG_PASSWORD";
-constexpr const char *kMgPassfile = "MG_PASSFILE";
+constexpr const char *kMgUser = "ENTERPRISE_MG_USER";
+constexpr const char *kMgPassword = "ENTERPRISE_MG_PASSWORD";
+constexpr const char *kMgPassfile = "ENTERPRISE_MG_PASSFILE";
```

Comment:
`MEMGRAPH_USER`
`MEMGRAPH_PASSWORD`
`MEMGRAPH_PASSFILE`

Reply:
@niko4299, somehow it wasn't clear again 😂 remove the `ENTERPRISE` part because this is not an enterprise feature 😄 so, I think the correct env var names should be:
`MEMGRAPH_USER`
`MEMGRAPH_PASSWORD`
`MEMGRAPH_PASSFILE`